libraries/lib-graphics/FrameStatistics.h

```diff
@@ -0,0 +1,92 @@
+/*  SPDX-License-Identifier: GPL-2.0-or-later */
+/**********************************************************************
+
+  Audacity: A Digital Audio Editor
+
+  FrameStatistics.h
+
+  Dmitry Vedenko
```

Comment:
Again `/*!` and use `@file` before the file name, and add a lest a `@brief` line about the purpose of the header.

Much more Doxygenation might be added but that much is the minimum I would like so there is some summary on pages like this one https://doxy.audacityteam.org/dir_68267d1309a1af8e8297ef4c3efbcdba.html

I'm less concerned to add `@file` to .cpp files, because they are not interfaces.  It's generally understood that they implement whatever the companion .h file is about.